cheroot/ssl/builtin.py

```diff
@@ -117,8 +150,56 @@ def get_environ(self, sock):
             # SSL_VERSION_INTERFACE     string  The mod_ssl program version
             # SSL_VERSION_LIBRARY   string  The OpenSSL program version
         }
+
+        client_cert = sock.getpeercert()
```

Comment:
I'd add `if self.verify_mod` check in order to avoid unnecessary syscall when this feature is disabled.